packages/@uppy/utils/src/VirtualList.tsx

```diff
@@ -65,58 +79,61 @@ class VirtualList extends Component {
     }
   }
 
-  componentDidMount () {
+  componentDidMount(): void {
     this.resize()
     window.addEventListener('resize', this.handleResize)
   }
 
   // TODO: refactor to stable lifecycle method
   // eslint-disable-next-line
-  componentWillUpdate () {
-    if (this.base.contains(document.activeElement)) {
-      this.focusElement = document.activeElement
+  componentWillUpdate() {
+    if (this.base!.contains(document.activeElement)) {
+      this.focusElement = document.activeElement as HTMLElement
     }
   }
 
-  componentDidUpdate () {
+  componentDidUpdate(): void {
```

Comment:
lot's unrelated changes aren't great in a PR this big. But we can leave it for now

Reply:
right. I touched on the code and then I noticed that we had forgotten to refactor it to TS, so I figured i might as well do it